charts/linkerd2-cni/templates/cni-plugin.yaml

```diff
@@ -18,22 +18,24 @@ This file was inspired by
 1) https://github.com/istio/cni/blob/c63a509539b5ed165a6617548c31b686f13c2133/deployments/kubernetes/install/manifests/istio-cni.yaml
 */ -}}
 {{- if eq .Release.Service "CLI" -}}
+---
 kind: Namespace
 apiVersion: v1
 metadata:
   name: {{.Release.Namespace}}
   labels:
     linkerd.io/cni-resource: "true"
     config.linkerd.io/admission-webhooks: disabled
----
 {{ end -}}
+---
```

Comment:
These couple of changes are the culprit of the failing CNI unit test, as the output in `cli/cmd/testdata/install_cni_helm_default_output.golden` and cli/cmd/testdata/install_cni_helm_override_output.golden` files got changed. Please undo this to avoid having the `---` delimiter being duplicated in the output.